doc/classes/Font.xml

```diff
@@ -248,6 +248,7 @@
 			</argument>
 			<description>
 				Returns the size size of a bounding box of a string, taking kerning and advance into account.
+				[b]Note:[/b] The height returned is the font height (see also [method get_height]) and has no relation to the string.
```

Comment:
This is wrong for master (it was this way in 3.2), returned height is not necessary equal to `get_height` of the font, since it takes vertical glyph offsets (nested diacritics, Nastaʼlīq) into account.

Reply:
Could you give me an example string to test with?
I tried 'نستعلیق' and it returned the same height of 23.

Reply:
The absolute majority of the real text will fit to the font height (most fonts are designed in this way), but it's possible:

```gdscript
	font = Font.new()
	var fd = load("res://NotoNastaliqUrdu-Regular.ttf")
	font.add_data(fd)
	
	print(font.get_height()) # prints 41
	print(font.get_string_size("نستعلیستعلیستعلیق")) # prints (94, 43)
```

<img width="102" alt="Screenshot 2021-03-17 at 16 34 30" src="https://user-images.githubusercontent.com/7645683/111485281-29dd7200-873f-11eb-8187-2a6a5ffb8b2d.png">

or 

```gdscript
	font = Font.new()
	var fd = load("res://NotoSansUI_Regular.ttf")
	font.add_data(fd)
	
	print(font.get_height()) # prints 23
	print(font.get_string_size("z̵̛̻̾̔͛͗̍̀̃̓̽a̷̛̺͚̩̞̪̋̎̊̅́́͌̊̈́́̓̔l̵̛͎͚̤͍̜̟͋̾̓̎̇̔̂̃̐̐̾̅̚g̷̛͇̼̫̙̳̖̳̥̩̫̘͙̈́̉̂͑̓̐̀͘̚ͅo̷̖̩̳̺̯̲͔͋̔̐͑̂̉̾͊̉̽͌̽̅̈̀́͠")) # prints (41, 113)
```

<img width="85" alt="Screenshot 2021-03-17 at 16 34 13" src="https://user-images.githubusercontent.com/7645683/111485302-2ea22600-873f-11eb-819b-3eeb76df8cbb.png">

Reply:
Thanks for that. Would it make sense to instead copy the note from `Font.get_height`?
>**Note:** Real height of the string is context-dependent and can be significantly different from the value returned by this function. Use it only as rough estimate (e.g. as the height of empty line).
Apologies if your proposed changes included that or a different suggestion, my force-push seems to have broken that so I can't see it.

Reply:
> Thanks for that. Would it make sense to instead copy the note from Font.get_height?

Yes, seems good.